lib/google_crawler/google/scrapper.ex

```diff
@@ -1,9 +1,39 @@
+defmodule GoogleCrawler.Google.ScrapperResult do
+  defstruct raw_html_result: nil,
+            total_results: 0
+end
+
 defmodule GoogleCrawler.Google.Scrapper do
+  alias GoogleCrawler.Google.ScrapperResult
```

Comment:
Where is `ScrapperResult` defined?

Reply:
@olivierobert In this file above here 😆 ⬆️

Reply:
Oh I see. Not sure about having more than one module per file though 🤔

Reply:
Agree 👍 I think it's better to separate it 🤔